src/api/migration/database.ts

```diff
@@ -0,0 +1,41 @@
+import { createConnection, Connection } from 'typeorm';
+import "reflect-metadata";
+
+class Database {
+   
+    public connection: Connection
+
+    constructor(
+        connection: Connection
+    ) {
+        this.connection = connection
+    }
+
+    public async connect(): Promise<void> {
+        this.connection = await createConnection({
+            "name": "db", 
+            "type": "sqlite" as any,
```

Comment:
@bigLucas
When I insert only "sqlite", it's generated a lot of errors across the object.
Researching I found a solution like: `"postgres" as "postgres"`, it works with some others databases, but not with sqlite.
Looking at PF project, I saw `"database" as any` and I used here too to avoid error.

**VSCode error message:**
○ No overload matches this call.
○   Overload 1 of 3, '(name: string): Promise<Connection>', gave the following error.
○   Argument of type '{ name: string; type: "sqlite"; host: string; port: number; username: string; password: string; database: string; synchronize: boolean; logging: boolean; entities: string[]; migrations: string[]; subscribers: string[]; }' is not assignable to parameter of type 'string'.
○   Overload 2 of 3, '(options: ConnectionOptions): Promise<Connection>', gave the following error.
○   Argument of type '{ name: string; type: "sqlite"; host: string; port: number; username: string; password: string; database: string; synchronize: true; logging: false; entities: string[]; migrations: string[]; subscribers: string[]; }' is not assignable to parameter of type 'ConnectionOptions'.
○   Object literal may only specify known properties, and '"host"' does not exist in type 'SqliteConnectionOptions'.ts(2769)

One of the discussion that helped me : https://github.com/typeorm/typeorm/issues/949

Reply:
@Aristevao 
I saw that `slite` is pretty different from the others database, [here](https://github.com/typeorm/typeorm/blob/master/docs/connection-options.md#sqlite-connection-options) you can see more information about the **connection options**.
I strongly recommend that you read this doc, to understand more about the options.
And, don't forget that with `any` you already solve the problem.

Reply:
@Aristevao another sample with SQLite: https://github.com/kentrino/typeorm-sqlite-example/blob/master/src/index.ts